src/Bicep.Cli/Arguments/FormatArguments.cs

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.IO.Abstractions;
 using Bicep.Core.FileSystem;
-using Bicep.Core.PrettyPrint.Options;
+using Bicep.Core.PrettyPrintV2;
 
 namespace Bicep.Cli.Arguments
 {
     public class FormatArguments : ArgumentsBase
     {
-        public FormatArguments(string[] args, IOContext io) : base(Constants.Command.Format)
+        public FormatArguments(string[] args, IOContext io, IFileSystem? fileSystem) : base(Constants.Command.Format)
```

Comment:
We should always pass in a file system

Reply:
Good call. Changed this to be a non-nullable type.